core/src/main/java/org/apache/gravitino/audit/AuditLogManager.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.audit;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.gravitino.exceptions.GravitinoRuntimeException;
+import org.apache.gravitino.listener.EventListenerManager;
+import org.apache.gravitino.listener.api.EventListenerPlugin;
+import org.apache.gravitino.listener.api.event.Event;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AuditLogManager {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AuditLogManager.class);
+
+  public static final String AUDIT_LOG_PREFIX = "gravitino.audit.";
+
+  @VisibleForTesting private AuditLogWriter auditLogWriter;
+
+  public void init(Map<String, String> properties, EventListenerManager eventBusManager) {
+    AuditLogConfig auditLogConfig = new AuditLogConfig(properties);
+    if (!auditLogConfig.isAuditEnabled()) {
+      LOG.warn("Audit log is not enabled");
+      return;
+    }
+
+    String writerClassName = auditLogConfig.getWriterClassName();
+    String formatterClassName = auditLogConfig.getAuditLogFormatterClassName();
+    Formatter formatter = loadFormatter(formatterClassName);
+    LOG.info("Audit log writer class name {}", writerClassName);
+    if (StringUtils.isEmpty(writerClassName)) {
```

Comment:
No need to check it again, it has been checked in ` auditLogConfig.getWriterClassName`